modules/dnn/src/layers/pooling_layer.cpp

```diff
@@ -287,7 +287,7 @@ class PoolingLayerImpl : public PoolingLayer
             bool compMaxIdx = computeMaxIdx;
 
 #if CV_SIMD128
-            const int* ofsptr = &ofsbuf[0];
+            const int* ofsptr = (const int*)ofsbuf;
```

Comment:
`const int* ofsptr = ofsbuf.empty() ? 0 : &ofsbuf[0];` is the better solution

Reply:
Perfect, I think it is OK.